spec/acceptance/suites/default/nodesets

```diff
@@ -0,0 +1 @@
+../../nodesets/
```

Comment:
You don't have to check in this link.  simp-beaker-helpers will create it for you on the fly